tools/sense_studio/templates/navigation.html

```diff
@@ -0,0 +1,66 @@
+{% block navigation %}
+    <a href="{{ url_for('project_details', project=project) }}" class="item">Setup Class</a>
```

Comment:
Can we have something like `Project Overview` instead of `Setup Class`? The latter is a bit unclear. Also, this may be addressed when we have a decent UI design to work with, so it's not a priority.